modify class IfAccessViewhelper to have an initializeArguments function

merzilla opened the pull request:
and so Interface to ceckAccess gets implemented.
This commit is an addition to you PR on https://github.com/mittwald/typo3_forum/pull/232

Commit:
[TASK] make default CSS optional

Commit:
[TASK] get limit for latest topics view from flexform

martin-helmich committed:
Merge remote-tracking branch 'origin/develop'

martin-helmich committed:
Merge remote-tracking branch 'origin/develop'

Commit:
[TASK] add default to flexform setting maxTopicItems

Commit:
[TASK] set default flexform setting maxTopicItems to 6

RamonaBurger committed:
Update setup.txt

Commit:
if glob() returns false (no array) than skip further foreach; fix for throw exception at second foreach()

syssi committed:
Template override of the pagination fixed.

syssi committed:
Last post reference fixed. Preserve the tx_mmforum_posts.uid instead of tx_mmforum_posts_text.uid.

syssi committed:
Field last_post_crdate of topics populated. Fixes the topic list order (TopicRepository->findForIndex).

syssi committed:
mm_forum migration: Migration of avatars added. Files needs to be moved to the new location manually.

martin-helmich committed:
Merge pull request #220 from syssi/develop

mm_forum migration: Topic list order fixed

martin-helmich committed:
Merge pull request #219 from syssi/develop

Last post reference fixed

martin-helmich committed:
Merge pull request #212 from syssi/upstream

Template override of the pagination fixed.

martin-helmich committed:
Merge pull request #210 from DonTermi/fixed-warning

fixes exception for CacheManager::deleteTemporaryFiles()

martin-helmich committed:
Merge pull request #221 from syssi/develop

mm_forum migration: Migration of avatars added.

twiesing committed:
[FIX] Composer install fails

martin-helmich committed:
Merge pull request #226 from thueske/develop

[FIX] Composer install fails

Commit:
[TASK] set requirement for static_info_tables to at least v6.3

Commit:
Switch to PSR-2 line indentation (4 spaces)

Commit:
Adding early return if recipient has no email.

Commit:
modify class to have an initializeArguments function and so Interface gets implemented

Commit:
Merge branch 'develop' of git://github.com/sventendo/typo3-forum into sventendo-develop

Commit:
#140 Add comment for editing optional flag in constant editor

Commit:
Merge branch 'sventendo-develop' into develop

kpurrmann committed:
Merge pull request #193 from 4youPromotionGmbHWerbeagentur/bugfix/imageUploadDirectory

change image directory to currently correct value

Commit:
#217 Add fixes for rank update

kpurrmann committed:
Merge pull request #233 from WebDevJona/develop

Don't send an email if FontendUser has no email adress set

Commit:
make class of rowViewHelper as argument, add serverSide redirect to new topic preview

Commit:
[TASK] add forwards to render preview server side by controllers. FIX in FavSubscribe, add setter.

merzilla committed:
Update composer.json

Add a version to mark as stable

merzilla committed:
Update composer.json

Rename package to original name of default branch

merzilla committed:
revert renaming

merzilla committed:
revert renaming

simonschaufi commented on composer.json:
```diff
@@ -3,6 +3,7 @@
 	"description": "Forum extension",
 	"license": "GPL-2",
 	"type": "typo3-cms-extension",
+    "version": "1.0.0",
```
a version is defined by the git tag, not by the version. Whats the reason adding it here?

merzilla replied:
I had problems with composer installing my fork and thought this could help.
Well, you can ignore this one, as it´s unnecessary. Sorry.

merzilla committed:
Merge pull request #1 from mittwald/develop

Get latest changes from develop

merzilla committed:
Merge branch 'verdure' into develop

merzilla committed:
Merge pull request #2 from merzilla/develop

Update verdure branch with current changes from original develop

Commit:
[FIX] Update inline TCA for topic posts. Add a foreign_table_name field. That´s also for v8 compatibility.

merzilla committed:
Update README.md

Fill new field `tablename` with default value.

Commit:
[FIX] set default value for tablename

merzilla committed:
remove version constraint

This was more a test and can be removed again.
According to comments on https://github.com/VERDURE-Medienteam/typo3_forum/pull/1#pullrequestreview-83588162

Commit:
auto-create attachments folder in uploads

ghost commented:
Hi Ralf,
sorry das Thema ist nicht vergessen. Bekomme das Thema aktuell bei uns nur zeitlich nicht unter da wir in dem Projekt wo das Forum in Einsatz ist andere Prios hat. Sobald ich die Sachen durchgetestet bekomme merge ich alles rein. 
Bei Mittwald selbst scheint das ganze aber aktuell leider auch auf unbestimmte Zeit zu pausieren, da ist fraglich ob die überhaupt was damit machen werden :(

Viele Grüße

merzilla commented:
Hi,

okay, danke für den Kommentar. Ja, scheint etwas zu pausieren das Ganze. Im Moment schaut's gut aus und wir haben das Forum soweit für unsere Bedürfnisse live. Soweit läuft´s bzw. sind jetzt eigentlich noch keine Bugs angeflogen gekommen.

Viele Grüße
Ralf

DavidBruchmann commented on Classes/Service/Mailing/HTMLMailingService.php:
```diff
@@ -43,9 +43,14 @@ class HTMLMailingService extends AbstractMailingService
+            return;
+        }
+
+        $typo3Mail = GeneralUtility::makeInstance('TYPO3\\CMS\\Core\\Mail\\MailMessage');
```
Why replacing `$typo3Mail = GeneralUtility::makeInstance(MailMessage::class);` by this again?

DavidBruchmann commented on Classes/ViewHelpers/Bootstrap/Form/RowViewHelper.php:
```diff
@@ -85,7 +87,7 @@ public function render() {
 		}
 
-		$label   = '<label>' . $label . '</label>';
+		$label   = '<h3>' . $label . '</h3>';
```
Here should be  some configuration IMO that the tags can be defined however liked

ghost commented:
We accept the PR. But to inform you all, the project which whe used the extension has been stopped a while ago. Since there wasn't any reaction at all from mittwald in continuing development, or even a reaction to this pr we won't actively work on this fork. i can personally look from time to time over pr's/comments but it will take some time to go through them.
